skops/card/_model_card.py

```diff
@@ -254,13 +297,28 @@ def __init__(
         model_diagram: bool = True,
         metadata: Optional[CardData] = None,
     ) -> None:
-        self.model = model
+        self._model = model
         self.model_diagram = model_diagram
         self._eval_results = {}  # type: ignore
         self._template_sections: dict[str, str] = {}
         self._extra_sections: list[tuple[str, Any]] = []
         self.metadata = metadata or CardData()
 
+    @property
+    def model(self):
+        model = _load_model(self._model)
+        if model is not self._model:
+            self._model = model
+        return model
+
+    @model.setter
+    def model(self, model):
+        self._model = model
+
+    @model.deleter
+    def model(self):
+        del self._model
```

Comment:
do we need this?

Reply:
For consistency, the method should exist (assuming we leave the rest as is), no?

Reply:
I almost never see a deleter for properties.